src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -352,6 +351,9 @@ public Builder runDevAppServerWait(int runDevAppServerWaitSeconds) {
 
     /**
      * Create a new instance of {@link CloudSdk}.
+     *
```

Comment:
for the moment get rid of this blank line until we can configure checkstyle to ignore this.